src/utils/transformationDescription.ts

```diff
@@ -104,6 +106,32 @@ export function useActiveTransformations(
     async function callback(deletedContext: string) {
       const cloned = { ...activeTransformations };
       for (const input of Object.keys(cloned)) {
+        // Partition transformations based on whether or not one of their input
+        // contexts was deleted
+        const transformationsWithNewlyMissingInputs = [];
+        const restOfTransformations = [];
+        for (const description of activeTransformations[input]) {
+          if (
+            description.inputs.includes(deletedContext) ||
+            description.extraDependencies.includes(deletedContext)
+          ) {
+            transformationsWithNewlyMissingInputs.push(description);
+          } else {
+            restOfTransformations.push(description);
+          }
+        }
+
+        // Rename transformations with newly missing inputs to add a [fixed] suffix
+        for (const transformation of transformationsWithNewlyMissingInputs) {
+          if ("output" in transformation) {
+            console.log("transformation", transformation);
+            const outputData = await getDataContext(transformation.output);
```

Comment:
I get an unhandled rejection for `getDataContext` when this is used with single-value transformers (I guess because they are not contexts but just text elements). Can you modify this to work with single-value transformers as well?

Reply:
Should be fixed now